docs/versioned/admin/editing-configmaps.md

```diff
@@ -0,0 +1,155 @@
+---
+audience: administrator
+components:
+  - serving
+  - eventing
+function: explanation
+---
+# Editing ConfigMaps
+
+This page provides information and best practices for editing ConfigMaps. While there is no technical difference in how Knative uses ConfigMaps in Kubernetes, you should be aware of the following points and tips while working with ConfigMaps:
+
+- The _example key. Don't modify.
+- Value propagation. Know when changes to default settings take affect.
+- Version control. Use ConfigMaps for version control.
+
+The most common way to use ConfigMaps is to configure settings for containers running in a Pod, or in multiple pods, that use a specified namespace. There are a number of other ways to use ConfigMaps as described in [Kubernetes ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/) documentation.
+
+## The _example key
+
+ConfigMap files installed by Knative contain an `_example` key that shows the usage and purpose of a configuration key. The key itself is not a key, but a long comment.
+
+In case a user edits the `_example` key by mistakenly thinking their edits would have an affect, the administrator needs to be alerted. The Knative webhook server determines if the `_example key` has been altered. The edit is caught when the value of the checksum for the `_example` key is different when compared with the pod's template annotations. If the checksum is null however, it does not create the warning.
+
+Accordingly, you cannot alter the contents of the `_example` key, but you can delete the `_example` key altogether or delete the annotation:
+
+```yml
+...
+annotations:
+    knative.dev/example-checksum: "47c2487f"
+```
+
+To accurately create a ConfigMap using an existing file, use the Kubernetes[Define the key to use when creating a configmap from a file](https://kubernetes.io/dos/tasks/configure-pod-container/configure-pod-configmap/#define-the-key-to-use-when-creating-a-configmap-from-a-file)
+
+## Value propagation
+
+The ConfigMap change is immediate in the Kubernetes API, but its effect on Pods or applications depends on how the ConfigMap is consumed (volume, environment variables, or API) and whether the application or Pod is designed to pick up the change dynamically or requires a restart. For volumes, updates propagate within seconds, but for environment variables, a Pod restart is needed.
```

Comment:
None of this applies to the Knative controllers, which use the Kubernetes Watch API to be informed of any changes to the ConfigMap roughly as soon as they happen.

Reply:
I'm not sure if any of this "value propagation" information is necessary given how Knative watches ConfigMaps directly.